files/zh-cn/web/javascript/guide/iterators_and_generators/index.md

```diff
@@ -56,41 +65,68 @@ while (!result.done) {
   result = it.next();
 }
 
-console.log("Iterated over sequence of size: ", result.value); // 5
+console.log(`已迭代序列的大小：${result.value}`); // 5
 ```
 
-> **备注：** 反射性地知道特定对象是否是迭代器是不可能的。如果你需要这样做，请使用 [Iterables](#Iterables).
+> **备注**：[反射性](https://zh.wikipedia.org/wiki/反射式编程)地知道特定对象是否是迭代器是不可能的。如果你需要这样做，请使用[可迭代对象](#可迭代对象).
```

Comment:
Please reserve this format, or the block will not be highlighted

![image](https://github.com/mdn/translated-content/assets/15844309/09eaa5c6-9e4c-468c-b0a8-d9e339b86b9d)

```suggestion
> **备注：** [反射性](https://zh.wikipedia.org/wiki/反射式编程)地知道特定对象是否是迭代器是不可能的。如果你需要这样做，请使用[可迭代对象](#可迭代对象)。
```